rocket-chat-android-widgets/src/main/res/layout/message_inline_attachment.xml

```diff
@@ -93,14 +96,12 @@
 
         </LinearLayout>
 
-        <ImageView
+        <com.facebook.drawee.view.SimpleDraweeView
                 android:id="@+id/image"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:maxHeight="200dp"
+                android:layout_width="match_parent"
+                android:layout_height="200dp"
                 android:layout_marginBottom="8dp"
-                android:adjustViewBounds="true"
-                android:scaleType="fitStart" />
+                fresco:actualImageScaleType="fitCenter" />
```

Comment:
I think `fitStart` is better here. 

Every image is put on the left-side in web-client
![image](https://cloud.githubusercontent.com/assets/11763113/22379457/c410cbc2-e4fc-11e6-8be6-40b17d7ec139.png)